courses/free-courses-en.md

```diff
@@ -273,6 +273,7 @@
 ### Compilers
 
 * [Compilers](https://www.youtube.com/playlist?list=PL6KMWPQP_DM97Hh0PYNgJord-sANFTI3i) - Ghassan Shobaki (YouTube)
+* [Neso's Compilers](https://youtube.com/playlist?list=PLBlnK6fEyqRjT3oJxFXRgjPNzeS-LFY-q) - Neso Academy (YouTube)
```

Comment:
Don't invent titles! Use the title given by the resource.

Reply:
Fixing as suggested by Eric...

```suggestion
* [Compiler Design](https://youtube.com/playlist?list=PLBlnK6fEyqRjT3oJxFXRgjPNzeS-LFY-q) - Neso Academy
```